src/util/index.js

```diff
@@ -0,0 +1,93 @@
+import path from 'path';
+import fs from 'fs';
+import Lumberjack, { monkeypatches } from '@americanexpress/lumberjack';
+
+const logger = new Lumberjack();
+monkeypatches.replaceGlobalConsole(logger);
```

Comment:
This should be handled by one app. is this required ?

Reply:
I am not sure.

Reply:
This library should not monkey patch the console

Reply:
@bishnubista can you remove this ?

Reply:
removed.